nextflow_schema.json

```
@@ -520,6 +520,11 @@
                     "type": "boolean",
                     "description": "Only return official taxonomic ranks (Kingdom, Phylum, etc.) when running CAT."
                 },
+                "cat_classify_unbinned": {
+                    "type": "boolean",
+                    "description": "Classify unbinned contigs with CAT (contig mode).",
+                    "help_text": "This will run CAT classification on all unbinned contigs, which can be useful to get an idea of the taxonomic composition of the unbinned contigs."
```

```suggestion
                    "help_text": "This will run CAT classification on all unbinned contigs, which can be useful to get an idea of the taxonomic composition of the unbinned contigs. This task might take a lot of time though."
```
Maybe the downside could be mentioned, if thats true.